packages/client/src/workflow-client.ts

```diff
@@ -946,6 +926,31 @@ export class WorkflowClient {
       followRuns: options?.followRuns ?? true,
     });
   }
+
+  /**
+   * List workflows by given `query`.
+   *
+   * See more info on how to build the query in the Temporal docs: https://docs.temporal.io/visibility
+   *
+   * This API requires Elasticsearch based visibility as of server version 1.18.
```

Comment:
Not exactly true as simple queries, e.g. listing open workflows, work w/out elastic search

Reply:
I didn't know that, I thought this entire API only works with ES right now.
I'll fix.

Reply:
Thanks, fixed.